src/relay/analysis/annotated_region_set.cc

```diff
@@ -76,17 +76,21 @@ void AnnotatedRegionSetNode::AddToRegion(AnnotatedRegion dest, const Expr& expr)
   }
 }
 
-AnnotatedRegion AnnotatedRegionSetNode::MakeRegion(const std::string& target) {
+AnnotatedRegion AnnotatedRegionSetNode::MakeRegion(const std::string& func_name,
+                                                   const std::string& target) {
   auto ret = regions_.emplace(AnnotatedRegion());
   (*ret.first)->id_ = region_id_++;
   (*ret.first)->target_ = target;
+  (*ret.first)->func_name_ = func_name;
   return *ret.first;
 }
 
 class AnnotatedRegionSet::Creator : protected MixedModeVisitor {
  public:
   Creator(const Op& region_begin_op, const Op& region_end_op)
       : begin_op_(region_begin_op), end_op_(region_end_op) {}
+  Creator(const Op& region_begin_op, const Op& region_end_op, const std::string& func_name)
+      : begin_op_(region_begin_op), end_op_(region_end_op), func_name(func_name) {}
```

Comment:
Is there any reason to keep both constructors?

Reply:
since it is being used here: https://github.com/apache/tvm/blob/73b38e894fcf4263eeb3978c85c2786218acaff8/src/relay/transforms/merge_compiler_regions.cc#L166

Reply:
Hmm I see. So in the case of merge compiler region we don't really care its name. How about keeping one constructor and assign a default value to func_name?

Reply:
agree, just addressed